docs/docs/08-working-with-the-browser.md

```diff
@@ -55,6 +55,6 @@ _Mounted_ composite components also support the following method:
 
 ## Browser Support
 
-React supports most popular browsers, including Internet Explorer 9 and above.
+React supports most popular browsers, including Internet Explorer 9 and above. After [React v15.0](https://facebook.github.io/react/blog/2016/04/07/react-v15.html), we've decided [not to actively support Internet Explorer 8](https://facebook.github.io/react/blog/2016/01/12/discontinuing-ie8-support.html) with any new fixes, as it is running fine in its current form.
```

Comment:
I don't think we want to say things like "as it is running fine in its current form." because this statement could become untrue at any time and without notice.  In fact, we might not even notice that we broke it, in which case we'd certainly forget to come back and update the docs.  The blog post did a pretty good job of explaining our position, and I generally think that's sufficient.  We generally don't get a lot of questions about this, so I'd say we should just leave it as-is.